### PR TITLE
vm: spec fixes

### PR DESCRIPTION
**Pull request by jsign**

### CODE REVIEW

File: core/vm/operations_verkle.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func gasExtCodeSize4762(evm *EVM, contract *Contract, stack *Stack, mem *Memory,
 
 func gasExtCodeHash4762(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
 	address := stack.peek().Bytes20()
-	if _, isPrecompile := evm.precompile(address); isPrecompile {
+	if _, isPrecompile := evm.precompile(address); isPrecompile || evm.isSystemContract(address) {
```
Comment (jsign) [inline core/vm/operations_verkle.go:64]: image

File: core/vm/operations_verkle.go [hunk 2]
```diff
@@ -132,6 +132,10 @@ func gasExtCodeCopyEIP4762(evm *EVM, contract *Contract, stack *Stack, mem *Memo
 		return 0, err
 	}
 	addr := common.Address(stack.peek().Bytes20())
+
+	if _, isPrecompile := evm.precompile(addr); isPrecompile {
```
Comment (jsign) [inline core/vm/operations_verkle.go:136]: image

Comment (jsign) [inline core/vm/operations_verkle.go:136]: Does L130 makes sense for precompiles? 
I see that [gasExtCodeCopyEIP2929](https://github.com/gballet/go-ethereum/blob/fbbb499dda8c70cae13e238d9209722465e0a033/core/vm/operations_acl.go#L125) doesn't make a distinction either, so I guess is coherent. Just wondering.

Comment (gballet) [reply core/vm/operations_verkle.go:136]: yeah, because the way `CODECOPY` works, if no code can be found, it will right-pad the data and fill RAM.

**Review by gballet — APPROVED** [review]
LGTM. I was a bit surprised to only see the 2935 contract in the list, but it makes sense, since this is the only one for Shanghai.

**Pull request merged**